ci/test/00_setup_env_native_fuzz_with_valgrind.sh

```diff
@@ -17,8 +17,4 @@ export FUZZ_TESTS_CONFIG="--valgrind"
 export GOAL="install"
 export BITCOIN_CONFIG="\
  -DBUILD_FOR_FUZZING=ON \
- -DSANITIZERS=fuzzer \
```

Comment:
Why not keep using clang for this job?

Reply:
Mmh, I was more concerned about losing the sanitizer, but I guess there is nothing really gained by having it turned on when running valgrind.